Assets/MixedRealityToolkit/Utilities/Physics/TwoHandScaleLogic.cs

```diff
@@ -25,34 +25,32 @@ public class TwoHandScaleLogic
         /// <summary>
         /// Initialize system with source info from controllers/hands
         /// </summary>
-        /// <param name="handsPressedMap">Dictionary that maps inputSources to states</param>
+        /// <param name="handsPressedArray">Dictionary that maps inputSources to states</param>
         /// <param name="manipulationRoot">Transform of gameObject to be manipulated</param>
```

Comment:
Nit: Update the doccomments (as per previous comment from last comment I just left - so meta - we should also document what this Vector3 should be -i.e. size two vector that has the old location in 0th index, new location in the 1th index)